frontend/src/components/Chat/chat.scss

```diff
@@ -0,0 +1,65 @@
+$message_background_color: #e6e6e6;
+$font_color: #606060;
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.input {
+    outline: none;
+    height: 25px;
+    border-radius: 30px;
+    left: 79.25%;
+    width: 18%;
+    position: fixed;
+    top: 95%;
+    padding-left: 10px;
+}
+
+::-webkit-scrollbar {
+    width: 10px;
+    left: -10px;
+}
+
+::-webkit-scrollbar-track {
+    background: #f1f1f1;
+}
+
+::-webkit-scrollbar-thumb {
+    background: #888;
+}
+
+::-webkit-scrollbar-thumb:hover {
+    background: #555;
+}
+
+ul {
```

Comment:
you should probably have a look at `bem` style of using css classes.  Also remember that you can nest classes in scss. You miss so much from scss magic.